plugin/network/cni/cni.go

```diff
@@ -0,0 +1,167 @@
+//
```

Comment:
The code looks fine, but I would prefer to move it under `network/cni` and not `plugin/network/cni`